Bottom: b4957a2e336132bbe0bbe06dd49db0ddba9bcf72
Top:    f482ccb80350f54e526eee4c9595c6b489b0052e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:21:05 +0000

w32: TMP Build fixes


---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 04aec82..f6a641c 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -2,9 +2,6 @@ NEVERBUILD="
 "
 
 NOBUILD="
-"
-
-BUILD="
 jpeg
 zlib
 libpng
@@ -12,6 +9,9 @@ tiff
 freetype
 pixman
 libiconv
+"
+
+BUILD="
 gettext
 glib
 atk
